v03_pipeline/lib/misc/sample_ids.py

```diff
@@ -0,0 +1,73 @@
+from collections import Counter
+
+import hail as hl
+
+
+class MatrixTableSampleSetError(Exception):
+    def __init__(self, message, missing_samples):
+        super().__init__(message)
+        self.missing_samples = missing_samples
+
+
+def vcf_remap(mt: hl.MatrixTable) -> hl.MatrixTable:
+    # TODO: add logic from Mike to remap vcf samples delivered from Broad WGS
+    return mt
+
+
+def remap_sample_ids(mt: hl.MatrixTable, project_remap_ht: hl.Table) -> hl.MatrixTable:
```

Comment:
These two functions are copied verbatim from the `luigi_pipeline`, _except_ I remove the line that removed variants with only reference data calls inside the `subset_samples` function.  Needed to keep the row (for sure) inside the `sample_lookup_table` and I felt we might as well keep the row inside the `variant_annotations_table` as well for simplicity.